packages/chan-ko-website/src/components/Hero/Hero.tsx

```diff
@@ -12,7 +12,9 @@ const Hero: React.FC = () => {
           <br></br>
           Empowering your Technological, Entertainment, and Edutainment Goals
         </p>
+        <a href="#services">
         <button className={styles.cta}>Discover Our Services</button>
+        </a>
```

Comment:
The action is occurring on a button and you wouldn't typically surround a button by an anchor (`a`).

But more importantly than what is typically done, your implementation creates behavior that differs from how other elements initiate navigation within the page (it switches directly to the page fragment without scrolling and it updates the URL in the web browser's address bar).

Let's use an `onClick` event handler like this:
```
<button onClick={e => handleScroll(e, 'services')} className={styles.cta}>
```

You will need to add a `handleScroll` method, similar to the one in the `Header` component. But rather than a `MouseEvent<HTMLAnchorElement>` the method for this component should use a `MouseEvent<HTMLButtonElement>` and it won't need to use `setIsMenuOpen` at all.